File: 2401-count-asterisks/2401-count-asterisks.cpp
class Solution {
public:
    int countAsterisks(string s) {
        int count = 0;
        bool ceck = true;
        for(int i =0;i<s.size();i++){
            if(s[i]=='|'){
                ceck = !ceck;
            }
            if(ceck==true && s[i]=='*'){
                count += 1;
            }
        }
        return count;
    }
};

File: 2401-count-asterisks/2401-count-asterisks.py
class Solution(object):
    def countAsterisks(self, s):
        """
        :type s: str
        :rtype: int
        """
        count = 0
        ceck = True
        for i in s:
            if i == '|':
                ceck = not ceck
            if ceck:    
                if i == '*':
                    count+= 1
        return count

        